src/Components/UserRepoInfo.jsx

```diff
@@ -0,0 +1,28 @@
+import React from 'react';
+
+class UserRepoInfo extends React.Component {
+    constructor(props){
+        super(props);
+    }
+
+    render() {
+        return (
+                <ul>
+                    {
+                        this.props.repo.map(element => {
+                            return (
+
+                                <li key={element.id}>
```

Comment:
Czy ładniej by to nie wyglądało, gdyby każdy element był Reactowym komponentem?

Reply:
oczywiście już poprawiam :)